WORKSPACE

```diff
@@ -221,7 +221,7 @@ filegroup(
     visibility = ["//visibility:public"],
 )
     """,
-    sha256 = "1c806e04ac5e3779032c06a6009350b3836b6809bb23812993d6ececd7047cf5",
+    #sha256 = "1c806e04ac5e3779032c06a6009350b3836b6809bb23812993d6ececd7047cf5",
```

Comment:
Why is this one commented out rather than replaced like the others?

Reply:
My fault. I forgot to update it. 60730262e0c05ea816d4482066f58298cc295e8c